src/main/java/com/sprints/swagger/SprintsSwagger.java

```diff
@@ -1,23 +1,23 @@
-package com.sprints.swagger;
-
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-
-import springfox.documentation.builders.PathSelectors;
-import springfox.documentation.builders.RequestHandlerSelectors;
-import springfox.documentation.spi.DocumentationType;
-import springfox.documentation.spring.web.plugins.Docket;
-import springfox.documentation.swagger2.annotations.EnableSwagger2;
-
-@Configuration
-@EnableSwagger2
-public class SprintsSwagger {
-	@Bean
-	public Docket api() {
-		return new Docket(DocumentationType.SWAGGER_2)
-				.select()
-				.apis(RequestHandlerSelectors.any())
-				.paths(PathSelectors.any())
-				.build();
-	}
-}
+package com.sprints.swagger;
```

Comment:
Why did you modified the swagger class ?

Reply:
Sorry about this, I don´t really remember touching anything from this file, I checked and for some reason Git shows it like I replaced the whole file with the same information that was already there before. I don´t know why that happened